composer.lock

```diff
@@ -81,6 +81,11 @@
                 "php": ">=5.3.2"
             },
             "type": "symfony-bundle",
+            "extra": {
```

Comment:
these differences are probably created by the composer version, or more generically from the difference of environment in which `composer` runs (the `composer` container image now)